app/routes/app._index.tsx

```diff
@@ -86,6 +86,7 @@ export const action = async ({ request }: ActionFunctionArgs) => {
 
 export default function Index() {
   const fetcher = useFetcher<typeof action>();
+  const buttonRef = useRef<HTMLButtonElement>(null);
```

Comment:
Is this the strategy we should be using for buttons on the title bar?
I couldn't find an example from the patterns documentation.